yaml/_dumper/dumper.ts

```diff
@@ -582,7 +582,7 @@ function writeFlowMapping(
 
     if (index !== 0) pairBuffer += ", ";
 
-    objectKey = objectKeyList[index];
+    objectKey = objectKeyList[index]!;
```

Comment:
Can we use `const [objectKey, objectValue] of objectKeyList` as the for-loop condition?

Reply:
Seems this can't be done because `objectKeyList` contains only the keys of `object`, so I think we can do `const [index, objectKey] of objectKeyList.entries()` and leave the `objectValue = object[objectKey]` inside the loop. Furthermore, `index` is needed in `(index !== 0) pairBuffer += ", "`